src/main/java/de/unistuttgart/iste/gits/flashcard_service/persistence/dao/FlashcardSideEntity.java

```diff
@@ -24,6 +24,9 @@ public class FlashcardSideEntity {
     @Column(nullable = false)
     private boolean isQuestion;
 
+    @Column(nullable = false)
```

Comment:
are you sure about the nullable false? If so, don't forget the schema migration

Reply:
yeah you're right. It's not really needed